Source/WebCore/rendering/FloatingObjects.cpp

```diff
@@ -439,6 +439,18 @@ LayoutUnit FloatingObjects::logicalRightOffset(LayoutUnit fixedOffset, LayoutUni
     return std::min(fixedOffset, adapter.offset());
 }
 
+void FloatingObjects::shiftFloatsBy(LayoutUnit blockShift)
+{
+    LayoutUnit shiftX = (m_horizontalWritingMode) ? 0_lu : -blockShift;
+    LayoutUnit shiftY = (m_horizontalWritingMode) ? blockShift : 0_lu;
+
+    for (auto it = m_set.begin(), end = m_set.end(); it != end; ++it) {
```

Comment:
Can we use a modern range loop here instead of iterators?

Reply:
I tried, I can't get it to work. `for (auto it : m_set)` doesn't compile... I'm not sure if that's because I don't understand the right syntax or if it legitimately doesn't know how to iterate like that... in any case this is at least consistent with the rest of the file?

Reply:
Can you try `for (auto& object : m_set)` ? If you still get an error, could you please let me know what the error is. It should work just fine. m_set is a ListHashSet AFAICT.

Reply:
Ok, fixed, thanks for your help!